alethzero/plugins/whisper/Whisper.cpp

```diff
@@ -53,6 +55,17 @@ static Public stringToPublic(QString const& _a)
 		return Public();
 }
 
+static shh::Topics stringToTopics(QString _s)
+{
+	shh::BuildTopic ret;
+	QStringList tx = _s.split("|", QString::SkipEmptyParts);
```

Comment:
This is a string containing a set of topics delimited by |.
You might need it if you want to send a single message with several topics.